### PR TITLE
Docs: Update `edge connector` information

### LINKED ISSUE #671: Add connector-edge as an optional dependency
Now that the [connector-edge](https://npmjs.com/packages/@sonarwhal/connector-edge) is published and working with RS2 and RS3 we should look into adding it as an optional dependency that will get installed if we detect a compatible Windows 10 version.

Things to do:
* [x] Add it as an optional dependency
* [x] Update the wizard to show it if it appears
* [ ] Update the documentation to mention it and list the limitations

### PR DESCRIPTION
**Pull request by molant**
Fix #671

## Pull request checklist

Make sure you:

- [X] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [X] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/contributing/pull-requests/#commitmessages)

For non-trivial changes, please make sure you also:

- [X] Added/Updated related documentation.

## Short description of the change(s)

Added the missing documentation

### CODE REVIEW

**Comment by molant** [comment]
@sarvaje something that I'm missing?
We should update the `connector-edge` to use the defaults for `tabUrl` , etc.

**Comment by molant** [comment]
@alrra, anything to add or can this be merged? Once @sarvaje updates `connector-edge` tomorrow Wednesday we (you) should do a release if possible 😇

File: docs/user-guide/connectors/index.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ The current supported connectors are:
 * `jsdom`: Your website will be loaded using [`jsdom`][jsdom].
 * `chrome`: Your website will be loaded using Chrome and the Chrome
   Debugging Protocol. This is one of the `remote-debugging-connector`s
+* `edge`: Your website will be loaded using Edge via the [`edge-diagnostics-adapter][eda].
+  You will need to run Windows 10 Creators Update or better to use it.
```
Comment (alrra) [inline docs/user-guide/connectors/index.md]: > or better

or later

File: docs/user-guide/connectors/index.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ The current supported connectors are:
 * `jsdom`: Your website will be loaded using [`jsdom`][jsdom].
 * `chrome`: Your website will be loaded using Chrome and the Chrome
   Debugging Protocol. This is one of the `remote-debugging-connector`s
+* `edge`: Your website will be loaded using Edge via the [`edge-diagnostics-adapter][eda].
```
Comment (alrra) [inline docs/user-guide/connectors/index.md]: > [`edge-diagnostics-adapter]

Add missing ` at the end.

**Comment by alrra** [comment]
> anything to add or can this be merged? 

Done.

> Once @sarvaje updates connector-edge tomorrow Wednesday we (you) should do a release if possible 😇

@sarvaje Let me know when that it's done.

File: docs/user-guide/connectors/index.md (hunk 1 above)
Comment (molant) [reply docs/user-guide/connectors/index.md]: Technically the Falls Creator Update is the best windows 10 version yet 😛

**Comment by sarvaje** [comment]
@alrra the PR for that is [here](https://github.com/sonarwhal/connector-edge/pull/10)

**Comment by alrra** [comment]
> he PR for that is here

@sarvaje Anything else that needs to be done, or can I publish a new version of `connector-edge
`, update the dependency here and publish also a new `sonarwhal` version?

**Comment by sarvaje** [comment]
Nothing else. You just need to do what you said :)

**Pull request closed without merging**